sdks/common/runner-api/pom.xml

```diff
@@ -65,14 +65,11 @@
         <artifactId>protobuf-maven-plugin</artifactId>
         <configuration>
           <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
-          <pluginId>grpc-java</pluginId>
```

Comment:
The protoc step runs too quickly and file locking on Windows causes the plugin to fail occasionally complaining of can't copy protoc.exe since the plugin copies protoc.exe to the target folder to launch it.